src/styles/styles.js

```diff
@@ -1042,7 +1042,7 @@ const styles = {
         textDecorationLine: 'none',
     },
 
-    singleEmojiText: {
+    multipleEmojisText: {
```

Comment:
Could rename it as`onlyEmojisText` or `emojisText`  as the style applies for only text with emojis, it doesn't matter single or multiple?

Reply:
> Could rename it as`onlyEmojisText` or `emojisText` as the style applies for only text with emojis, it doesn't matter single or multiple?

Updated